src/core/AnimatedBezier.js

```diff
@@ -64,7 +64,7 @@ function newtonRaphsonIterate(aX, aGuessT, mX1, mX2) {
 }
 
 function bezier(mX1, mY1, mX2, mY2) {
-  if (!(0 <= mX1 && mX1 <= 1 && 0 <= mX2 && mX2 <= 1)) {
+  if (!(mX1 >= 0 && mX1 <= 1 && mX2 >= 0 && mX2 <= 1)) {
```

Comment:
I think `eslint-disable-line yoda` must have been above the if condition before. `eslint --fix` changed it to this, I don't see any reason not to prefer the non-yoda version so I'll just remove `eslint-disable-line yoda` and use it.

Reply:
Yoda version is cleaner IMO as it look closer to `0 <= mX1 <= 1` which is essentially what we want to check here, just need to have two comparisons because JS (and most programming languages) doesn't allow this type of syntax